ios/FluentUI/TextField/FluentTextField.swift

```diff
@@ -0,0 +1,300 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import Combine
+
+@objc(MSFTextField)
+public final class FluentTextField: UIView, UITextFieldDelegate, TokenizedControlInternal {
+    public override func didMoveToWindow() {
```

Comment:
FYI We're changing all FUA views to do tokenSet updates during `willMove(toWindow:)` -- could you update this too?